src/main/java/Ui.java

```diff
@@ -0,0 +1,17 @@
+public class Ui {
```

Comment:
I like the way you introduced a new class to house the logo and the UI of your program.